services/director/Dockerfile

```diff
@@ -55,11 +55,11 @@ RUN $SC_PIP install --upgrade \
 WORKDIR /build
 
 # install base 3rd party dependencies
-COPY  services/director/requirements/*.txt \
-      services/director/requirements/
+COPY  --chown=scu:scu services/director/requirements/*.txt \
+      tmp/director/requirements/
 
 RUN $SC_PIP install \
-      -r services/director/requirements/base.txt
+      -r tmp/director/requirements/base.txt
```

Comment:
How does this avoid docker to invalidate cache? Isn't it now copied to ``/build/tmp/director/requirements``  instead of ``/build/services/director/requirements`` 

The ``--chown=scu:scu`` might have an effect since host user might have different user id as container

Reply:
Yes, that's the part I'm not yet sure. But about 10/20 lines later there is again a COPY call that copies the entire services/director folder onto the same folder inside the container. I am still not sure if that is the issue in the end. the __--chown__ I added because the subsequent copy uses it, so I was wondering if this was actually invalidating the cache.